test/evergreen_develop.yml

```diff
@@ -71,6 +71,71 @@ buildvariants:
       execution_tasks:
       - ".wt2853-perf"
 
+- name: ubuntu2004-perf-tests-arm64
+  display_name: Ubuntu 20.04 Performance tests (ARM64)
+  batchtime: 1440 # 1 day
+  run_on:
+    - ubuntu2004-arm64-large
+  expansions:
+    test_env_vars:
+      WT_TOPDIR=$(git rev-parse --show-toplevel)
+      WT_BUILDDIR=$WT_TOPDIR/cmake_build
+      LD_LIBRARY_PATH=$WT_BUILDDIR:$WT_BUILDDIR/test/utility/
+    CC_OPTIMIZE_LEVEL: -DCC_OPTIMIZE_LEVEL=-O3
+    HAVE_DIAGNOSTIC: -DHAVE_DIAGNOSTIC=0
+    CMAKE_TOOLCHAIN_FILE: -DCMAKE_TOOLCHAIN_FILE=../cmake/toolchains/mongodbtoolchain_stable_gcc.cmake
+    CMAKE_INSTALL_PREFIX: -DCMAKE_INSTALL_PREFIX=$(pwd)/cmake_build/LOCAL_INSTALL
+    python_binary: '/opt/mongodbtoolchain/v4/bin/python3'
+    pip3_binary: '/opt/mongodbtoolchain/v4/bin/pip3'
+    virtualenv_binary: '/opt/mongodbtoolchain/v4/bin/virtualenv'
+    smp_command: -j $(echo "`grep -c ^processor /proc/cpuinfo` * 2" | bc)
+    cmake_generator: Ninja
+    make_command: ninja
+    collection: AllPerfTestsARM64
+  tasks:
+    - name: compile
+    - name: ".btree-perf"
+    # Disable LSM perf tests till the support for LSM is restored.
```

Comment:
I know this comment was present already but could we add a FIXME-WT-8867 to all the comments mentioning LSM being reenabled?

Reply:
Updated in [eff9519](https://github.com/wiredtiger/wiredtiger/pull/8909/commits/eff95194e797b614fcb33c3dbea59df5256a9fbf)